src/components/application_manager/src/mobile_command_factory.cc

```diff
@@ -143,7 +143,6 @@ CommandSharedPtr MobileCommandFactory::CreateCommand(
     commands::Command::CommandOrigin origin,
     ApplicationManager& application_manager) {
   CommandSharedPtr command;
-
```

Comment:
@IGapchuk unrelated changes

Reply:
@AKalinich-Luxoft Reverted in 36b0eadff97e5756477026c3d51f33e575f5d77a.